.golangci.yaml

```diff
@@ -1,66 +1,94 @@
-      - shadow
-    settings:
-      printf:
-        funcs:
-          - (github.com/databricks/cli/internal/testutil.TestingT).Infof
-          - (github.com/databricks/cli/internal/testutil.TestingT).Errorf
-          - (github.com/databricks/cli/internal/testutil.TestingT).Fatalf
-          - (github.com/databricks/cli/internal/testutil.TestingT).Skipf
-  gofmt:
-    rewrite-rules:
-      - pattern: 'a[b:len(a)]'
-        replacement: 'a[b:]'
-      - pattern: 'interface{}'
-        replacement: 'any'
-  errcheck:
-    exclude-functions:
-      - (*github.com/spf13/cobra.Command).RegisterFlagCompletionFunc
-      - (*github.com/spf13/cobra.Command).MarkFlagRequired
-      - (*github.com/spf13/pflag.FlagSet).MarkDeprecated
-      - (*github.com/spf13/pflag.FlagSet).MarkHidden
-  gofumpt:
-    module-path: github.com/databricks/cli
-    extra-rules: true
-  testifylint:
-    enable-all: true
-    disable:
-      # good check, but we have too many assert.(No)?Errorf? so excluding for now
-      - require-error
-  copyloopvar:
-    check-alias: true
-  gocritic:
-    disable-all: true
-    enabled-checks:
-      - ruleguard
-    settings:
-      ruleguard:
-        rules: 'libs/gorules/rule_*.go'
-        failOn: all
+    - unused
+  settings:
+    copyloopvar:
+      check-alias: true
+    errcheck:
+      exclude-functions:
+        - (*github.com/spf13/cobra.Command).RegisterFlagCompletionFunc
+        - (*github.com/spf13/cobra.Command).MarkFlagRequired
+        - (*github.com/spf13/pflag.FlagSet).MarkDeprecated
+        - (*github.com/spf13/pflag.FlagSet).MarkHidden
+    gocritic:
+      disable-all: true
+      enabled-checks:
+        - ruleguard
+      settings:
+        ruleguard:
+          failOn: all
+          rules: libs/gorules/rule_*.go
+    govet:
+      disable:
+        - fieldalignment
+        - shadow
+      enable-all: true
+      settings:
+        printf:
+          funcs:
+            - (github.com/databricks/cli/internal/testutil.TestingT).Infof
+            - (github.com/databricks/cli/internal/testutil.TestingT).Errorf
+            - (github.com/databricks/cli/internal/testutil.TestingT).Fatalf
+            - (github.com/databricks/cli/internal/testutil.TestingT).Skipf
+    staticcheck:
+      checks:
+        - all
+        # disabled checks:
+        - -ST1003
+        - -ST1016
+        - -ST1005
+        - -ST1023
+        - -QF1001
+        - -QF1003
+        - -QF1007
+        - -QF1008
+        - -QF1009
+        - -QF1011
+        - -QF1012
+    testifylint:
+      enable-all: true
+      disable:
+        - require-error # good check, but we have too many assert.(No)?Errorf? so excluding for now
+        - empty
+        - len
+        - equal-values
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
 issues:
-  exclude-dirs-use-default: false  # recommended by docs https://golangci-lint.run/usage/false-positives/
```

Comment:
auto migration just dropped this.
i could not find that this is recommended anymore?